src/parser/jobs/sge/modules/Gauge.tsx

```diff
@@ -95,22 +96,35 @@ export class Gauge extends CoreGauge {
 	override initialise() {
 		super.initialise()
 
+		// Initial filter setup
 		const playerFilter = filter<Event>().source(this.parser.actor.id)
 		const partyIds = this.actors.friends.filter(friend => friend.playerControlled).map(friend => friend.id)
 		const partyFilter = filter<Event>().target(oneOf(partyIds))
 		const playerDiagnosisPartyFilter = partyFilter.source(this.parser.actor.id).status(this.data.statuses.EUKRASIAN_DIAGNOSIS.id)
+		const playerPrognosisSelfFilter = playerFilter.target(this.parser.actor.id).status(this.data.statuses.EUKRASIAN_PROGNOSIS.id)
+
+		// Hook Addersgall consumer and generator actions
 		this.addEventHook(playerFilter.type('action').action(this.data.matchActionId(ADDERSGALL_CONSUMERS)), this.onConsumeAddersgall)
 		this.addEventHook(playerFilter.action(this.data.actions.RHIZOMATA.id), this.onRhizomata)
 
 		// Hook shield applications/removals that could generate Addersting
 		this.addEventHook(playerDiagnosisPartyFilter.type('statusApply'), this.onShieldApply)
 		this.addEventHook(playerDiagnosisPartyFilter.type('statusRemove'), this.onShieldRemove)
+		// For 6.3 and later patches, self-targeted Prognosis can generate Addersting
+		if (!this.parser.patch.before('6.3')) {
```

Comment:
`!before` feels like it should be `after`, but i realise that `after('6.3')` isn't actually going to do what we want here.

looking at the patch utility, i'm... tempted to say it needs a bit of a re-think, or at least extension - comparing `patch <-> patch` doesn't really make sense - in common vernacular, we reference patches as the point in time of their introduction, and it would be accurate to say "a fight logged right now is logged after 6.3)

maybe the comparisons can accept either a patch or a timestamp, and can then act as-current on patches (which isn't _incorrect_, semantically), and act as point-in-time for timestamps. idk. thoughts.